gravitee-am-ui/src/app/components/navbar/navbar.component.html

```diff
@@ -21,12 +21,12 @@
       <button mat-icon-button gioTopBarHome>
         <mat-icon svgIcon="gio:gravitee" (click)="navigateToHome()"></mat-icon>
       </button>
-      <gio-top-bar-content type="am" productName="Access Management"></gio-top-bar-content>
+      <gio-top-bar-content type="apim" productName="API Management"></gio-top-bar-content>
```

Comment:
**question**: is it intentional ?

Reply:
oups nope. error on my part i had a problem with the height and i tested with the APIM one. sorry